improver/blending/weights.py

```diff
@@ -231,36 +239,29 @@ def _check_config_dict(self):
                 )
                 raise ValueError(msg)
 
-    def _get_interpolation_inputs_from_dict(self, cube):
+    def _get_interpolation_inputs_from_dict(
+        self, cube: Cube
+    ) -> Tuple[ndarray, ndarray, ndarray, Tuple[int, int]]:
         """
         Generate inputs required for linear interpolation.
 
         Args:
-            cube (iris.cube.Cube):
+            cube:
                 Cube containing the coordinate information that will be used
                 for setting up the interpolation inputs.
 
         Returns:
-            (tuple): tuple containing:
-                **source_points** (numpy.ndarray):
-                    Points within the configuration dictionary that will
-                    be used as the input to the interpolation.
-
-                **target_points** (numpy.ndarray):
-                    Points within the cube that will be the target points
-                    for the interpolation.
-
-                **source_weights** (numpy.ndarray):
-                    Weights from the configuration dictionary that will be
-                    used as the input to the interpolation.
-
-                **fill_value** (tuple):
-                    Values that be used if extrapolation is required. The
-                    fill values will be used as constants that are extrapolated
-                    if the target_points are outside the source_points
-                    provided. These are equal to the first and last values
-                    provided by the source weights.
-
+            - Points within the configuration dictionary that will
+              be used as the input to the interpolation.
+            - Points within the cube that will be the target points
+              for the interpolation.
+            - Weights from the configuration dictionary that will be
+              used as the input to the interpolation.
+            - Values that be used if extrapolation is required. The
+              fill values will be used as constants that are extrapolated
+              if the target_points are outside the source_points
+              provided. These are equal to the first and last values
+              provided by the source weights.
```

Comment:
I think it's outside of the scope of the PR to make changes to the content of the docstrings. I've put the returns into a list format so that sphinx-autodoc-typehints picks up the types properly.

Reply:
Was referring to the headings that were in the old docs (so not changing content), maybe they could be included in the list at the beginning? Also fine if not.